devops/openshift/web-deploy.yml

```diff
@@ -97,6 +94,17 @@ objects:
       minReplicas: "${{REPLICAS}}"
       maxReplicas: 10
       targetCPUUtilizationPercentage: 80
+  - apiVersion: policy/v1
+    kind: PodDisruptionBudget
+    metadata:
+      name: ${NAME}-pdb
+    spec:
+      selector:
+        matchLabels:
+          app: ${NAME}
+      minAvailable: 2
```

Comment:
Correct me if wrong, I see the kubernetes documentation suggesting differently for frontends.

![image](https://github.com/bcgov/SIMS/assets/54600590/39389fd2-93b2-459a-911e-f42f087923a2)

Reply:
In the documentation you have shared the minAvailable should not be reduced by the serving capacity by more than 10%. In our scenario the HorizontalPodAutoscaler we have a maxReplica of 10 and PDB - minAvailable as 2, this means the overall reducing capacity is still at 20% which actually follows the documentation.